package.json

```diff
@@ -35,8 +36,8 @@
     "buffer-more-ints": "0.0.2"
   },
   "devDependencies": {
-    "pegjs": "0.9.0",
-    "ut-tools": "^5.13.5"
+    "pegjs": "0.10.0",
```

Comment:
Just note, that updating pegjs will not have any effect, until parser is not recompiled with it.